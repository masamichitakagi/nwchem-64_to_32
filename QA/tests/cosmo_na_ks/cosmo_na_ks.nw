start cosmo_na_dat
echo
title "Na+"
geometry
  na 0.0 0.0 0.0
end
charge +1
basis
  na library 6-31g*
end
cosmo
  do_cosmo_ks
end
scf
  print cosmo_energies
end
task scf energy
