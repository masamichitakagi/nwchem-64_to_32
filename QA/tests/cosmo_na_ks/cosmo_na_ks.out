 argument  1 = cosmo_na_ks.nw
All connections between all procs tested: SUCCESS
 


============================== echo of input deck ==============================
start cosmo_na_dat
echo
title "Na+"
geometry
  na 0.0 0.0 0.0
end
charge +1
basis
  na library 6-31g*
end
cosmo
  do_cosmo_ks
end
scf
  print cosmo_energies
end
task scf energy
================================================================================


                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                              Copyright (c) 1994-2013
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute
 
             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file
 
                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = arcen
    program         = ../../../bin/LINUX64/nwchem
    date            = Sat Oct 18 23:15:02 2014

    compiled        = Sat_Oct_18_22:06:41_2014
    source          = /home/d3y133/nwchem-dev/nwchem-newest
    nwchem branch   = Development
    nwchem revision = 26300
    ga revision     = 10506
    input           = cosmo_na_ks.nw
    prefix          = cosmo_na_dat.
    data base       = ./cosmo_na_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107194 doubles =    100.0 Mbytes
    stack    =   13107199 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428793 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------
 
  0 permanent = .
  0 scratch   = .
 
 
 
 
                                NWChem Input Module
                                -------------------
 
 
                                        Na+
                                        ---

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 na                  11.0000     0.00000000     0.00000000     0.00000000
 
      Atomic Mass 
      ----------- 
 
      na                22.989800
 

 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
 
            XYZ format geometry
            -------------------
     1
 geometry
 na                    0.00000000     0.00000000     0.00000000
 
                      Basis "ao basis" -> "" (cartesian)
                      -----
  na (Sodium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  9.99320000E+03  0.001938
  1 S  1.49989000E+03  0.014807
  1 S  3.41951000E+02  0.072706
  1 S  9.46797000E+01  0.252629
  1 S  2.97345000E+01  0.493242
  1 S  1.00063000E+01  0.313169
 
  2 S  1.50963000E+02 -0.003542
  2 S  3.55878000E+01 -0.043959
  2 S  1.11683000E+01 -0.109752
  2 S  3.90201000E+00  0.187398
  2 S  1.38177000E+00  0.646699
  2 S  4.66382000E-01  0.306058
 
  3 P  1.50963000E+02  0.005002
  3 P  3.55878000E+01  0.035511
  3 P  1.11683000E+01  0.142825
  3 P  3.90201000E+00  0.338620
  3 P  1.38177000E+00  0.451579
  3 P  4.66382000E-01  0.273271
 
  4 S  4.97966000E-01 -0.248503
  4 S  8.43530000E-02 -0.131704
  4 S  6.66350000E-02  1.233520
 
  5 P  4.97966000E-01 -0.023023
  5 P  8.43530000E-02  0.950359
  5 P  6.66350000E-02  0.059858
 
  6 S  2.59544000E-02  1.000000
 
  7 P  2.59544000E-02  1.000000
 
  8 D  1.75000000E-01  1.000000
 


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 na                          6-31g*                  8       19   4s3p1d


 
 solvent parameters
 solvname_short: h2o     
 solvname_long:  water                              
 dielec:       78.4000
 dielecinf:     1.7769

          ---------------
          -cosmo- solvent
          ---------------
 Cosmo: Klamt-Schuurmann, doi: 10.1039/P29930000799
 dielectric constant -eps-     =  78.40
 screen = (eps-1)/(eps+1/2)    =   0.98099
 surface charge correction     = scale
 -lineq- algorithm             =   0
 -bem- low  level              =   2
 -bem- high level              =   4
 -bem- from -octahedral-
 solvent radius (ang.)         =   0.500
 atomic radii = 
 --------------
    1 11.000  1.755

 solvent accessible surface
 --------------------------

 ---------- ATOMIC COORDINATES (A.U.) ------------ VDWR(ANG.) --
     1    0.00000000    0.00000000    0.00000000     1.755
 number of segments per atom =         32
 number of   points per atom =        640
 atom (   nspa,  nppa )
 ----------------------
    1 (     32,   640 )     640
 number of -cosmo- surface points =       32
 molecular surface =     38.705 angstrom**2
 molecular volume  =     22.642 angstrom**3
 G(cav/disp)       =      1.054 kcal/mol
 ...... end of -cosmo- initialization ......


                                 NWChem SCF Module
                                 -----------------
 
 
                                        Na+
 
 

  ao basis        = "ao basis"
  functions       =    19
  atoms           =     1
  closed shells   =     5
  open shells     =     0
  charge          =   1.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./cosmo_na_dat.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 na                          6-31g*                  8       19   4s3p1d



 Forming initial guess at       0.0s

 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:        -161.84139203

 Renormalizing density from      11.00 to     10
 
      Non-variational initial energy
      ------------------------------

 Total energy =    -152.531239
 1-e energy   =    -207.048349
 2-e energy   =      54.517110
 HOMO         =      -2.550013
 LUMO         =      -0.357699
 

 Starting SCF solution at       0.1s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------

     COSMO gas phase

 #quartets = 6.660D+02 #integrals = 3.754D+03 #direct =  0.0% #cached =100.0%


 Integral file          = ./cosmo_na_dat.aoints.0
 Record size in doubles =    65536    No. of integs per rec  =    43688
 Max. records in memory =        2    Max. records in file   = ********
 No. of bits per label  =        8    No. of bits per value  =       64


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -161.6427062967  4.80D-01  2.07D-01      0.2
                 2     -161.6591185609  4.89D-02  1.78D-02      0.2
                 3     -161.6592883161  8.44D-04  3.19D-04      0.2
                 4     -161.6592883669  9.05D-07  5.75D-07      0.2
     COSMO solvation phase
 Alternative 1
 -atmefc- energy =      -3.253723825042
 -elcefc- energy =       2.957910547641
 -efcefc- energy =       0.147887038068
 -lambda- energy =       0.000000000000
 -solnrg- energy =      -0.147926239333
 -allefc- energy =      -0.295813277401     -0.295774076136
 -ecos  - energy =      -3.105836786974
 Alternative 2
 -atmefc- energy =      -3.253723825042
 -elcefc- energy =       2.957910547641
 -allefc- energy =      -0.295813277401
 -solnrg- energy =      -0.147906638700
 -ecos  - energy =      -3.105817186341

              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -161.8071950056  2.34D-03  1.20D-03      0.2
 Alternative 1
 -atmefc- energy =      -3.253723825042
 -elcefc- energy =       2.957909616012
 -efcefc- energy =       0.147887038068
 -lambda- energy =       0.000000000000
 -solnrg- energy =      -0.147927170961
 -allefc- energy =      -0.295814209029     -0.295774076136
 -ecos  - energy =      -3.105836786974
 Alternative 2
 -atmefc- energy =      -3.253723825042
 -elcefc- energy =       2.957909616012
 -allefc- energy =      -0.295814209029
 -solnrg- energy =      -0.147907104515
 -ecos  - energy =      -3.105816720527
                 2     -161.8071949947  2.09D-06  1.21D-06      0.2
 
                  COSMO solvation results
                  -----------------------
 
                 gas phase energy =      -161.6592883669
                 sol phase energy =      -161.8071949947
 (electrostatic) solvation energy =         0.1479066278 (   92.81 kcal/mol)


       Final RHF  results 
       ------------------ 

         Total SCF energy =   -161.807194994698
      One-electron energy =   -221.787119585122
      Two-electron energy =     63.085741310951
 Nuclear repulsion energy =      0.000000000000
             COSMO energy =     -3.105816720527

        Time for solution =      0.1s


             Final eigenvalues
             -----------------

              1      
    1  -40.4848
    2   -2.7890
    3   -1.5113
    4   -1.5113
    5   -1.5113
    6    0.0294
    7    0.0571
    8    0.0571
    9    0.0571
   10    0.1560
   11    0.2541
   12    0.2541
   13    0.2541
   14    0.5186
   15    0.5186
 
                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-2.788988D+00
              MO Center=  1.5D-17, -2.1D-16,  1.6D-17, r^2= 2.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.028271  1 Na s                  1     -0.245818  1 Na s          
 
 Vector    3  Occ=2.000000D+00  E=-1.511331D+00
              MO Center= -1.3D-18,  2.2D-18, -1.0D-17, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.993446  1 Na pz         
 
 Vector    4  Occ=2.000000D+00  E=-1.511331D+00
              MO Center= -1.1D-17,  1.0D-17,  2.3D-18, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.993472  1 Na px         
 
 Vector    5  Occ=2.000000D+00  E=-1.511331D+00
              MO Center=  3.9D-18,  1.4D-16,  4.2D-18, r^2= 2.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.998834  1 Na py         
 
 Vector    6  Occ=0.000000D+00  E= 2.936071D-02
              MO Center=  6.1D-15, -5.1D-16,  3.2D-15, r^2= 1.1D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      1.576746  1 Na s                  6     -1.045571  1 Na s          
     2     -0.210800  1 Na s                 14      0.202940  1 Na dxx        
    17      0.202940  1 Na dyy               19      0.202940  1 Na dzz        
 
 Vector    7  Occ=0.000000D+00  E= 5.708555D-02
              MO Center= -4.4D-16, -6.0D-16, -3.6D-15, r^2= 1.5D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    13      1.085924  1 Na pz                12      0.176785  1 Na py         
     9     -0.175715  1 Na pz                11      0.166243  1 Na px         
 
 Vector    8  Occ=0.000000D+00  E= 5.708555D-02
              MO Center= -2.6D-16,  1.4D-15, -2.0D-16, r^2= 1.5D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    12      1.098277  1 Na py                13     -0.178627  1 Na pz         
     8     -0.177713  1 Na py         
 
 Vector    9  Occ=0.000000D+00  E= 5.708555D-02
              MO Center= -6.0D-15,  4.0D-17,  7.3D-16, r^2= 1.5D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.100219  1 Na px                 7     -0.178028  1 Na px         
    13     -0.164263  1 Na pz         
 
 Vector   10  Occ=0.000000D+00  E= 1.560407D-01
              MO Center=  2.5D-15,  2.2D-16,  1.9D-15, r^2= 1.1D+01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      4.220475  1 Na s                 10     -2.231658  1 Na s          
    14     -0.756387  1 Na dxx               17     -0.756387  1 Na dyy        
    19     -0.756387  1 Na dzz        
 
 Vector   11  Occ=0.000000D+00  E= 2.541217D-01
              MO Center=  1.1D-15, -1.1D-15, -1.4D-15, r^2= 6.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      1.050283  1 Na pz                 8      0.872080  1 Na py         
    13     -0.597471  1 Na pz                12     -0.496098  1 Na py         
     5     -0.165622  1 Na pz         
 
 Vector   12  Occ=0.000000D+00  E= 2.541217D-01
              MO Center=  1.7D-16,  8.8D-16,  6.5D-17, r^2= 6.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.052073  1 Na py                 9     -0.859697  1 Na pz         
    12     -0.598490  1 Na py                13      0.489053  1 Na pz         
     4     -0.165904  1 Na py                 7      0.160876  1 Na px         
 
 Vector   13  Occ=0.000000D+00  E= 2.541217D-01
              MO Center= -2.2D-15, -7.2D-16, -4.1D-16, r^2= 6.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.355631  1 Na px                11     -0.771174  1 Na px         
     3     -0.213773  1 Na px                 9      0.172176  1 Na pz         
 
 Vector   14  Occ=0.000000D+00  E= 5.186155D-01
              MO Center= -4.9D-16, -8.0D-18, -1.5D-16, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      1.685646  1 Na dxz               15     -0.386999  1 Na dxy        
 
 Vector   15  Occ=0.000000D+00  E= 5.186155D-01
              MO Center= -4.8D-16,  1.5D-15, -5.6D-16, r^2= 2.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      1.400784  1 Na dyz               15      1.007151  1 Na dxy        
    16      0.153134  1 Na dxz        
 

 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.000000000000           0.000000000000
 
  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 Na  11    10.00   2.00  2.00  5.99  0.00  0.01  0.00 -0.00 -0.00
 
       Multipole analysis of the density wrt the origin
       ------------------------------------------------
 
     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      1.000000      0.000000     11.000000
 
     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1     -0.000000      0.000000      0.000000
 
     2   2 0 0     -2.153339      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -2.153339      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -2.153339      0.000000      0.000000
 

 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.2s     wall:        0.2s
 
 
                                NWChem Input Module
                                -------------------
 
 
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  200      200     1506      884      792        0        0       36     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             7.76e+05 5.06e+05 1.17e+05 0.00e+00 0.00e+00 2.88e+02
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 39432 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        21	        23
	current total bytes		         0	         0
	maximum total bytes		   1065608	  22509400
	maximum total K-bytes		      1066	     22510
	maximum total M-bytes		         2	        23
 
 
                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:
 
                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018
 
                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
        S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen, V. Anisimov,
    F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus, M. Malagoli, A. Marenich,
   A. Otero-de-la-Roza, J. Mullin, P. Nichols, R. Peverati, J. Pittner, Y. Zhao,
         P.-D. Fan, A. Fonari, R. J. Harrison, M. Dupuis, D. Silverstein,
    D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.2s     wall:        0.2s
